src/wizard/dotnet/maui.md

```diff
@@ -0,0 +1,104 @@
+---
+name: Multi-platform App UI (MAUI)
+doc_link: https://docs.sentry.io/platforms/dotnet/guides/maui/
+support_level: production
+type: framework
+---
+
+Install the **NuGet** package:
+
+.NET Core CLI:
+
+```shell
+dotnet add package Sentry.Maui -v 3.19.0-preview.2
+```
+
+Or, with the Visual Studio Package Manager:
+
+```powershell
+Install-Package Sentry.Maui -Version 3.19.0-preview.2
+```
+
+Then add Sentry to `MauiProgram.cs` through the `MauiAppBuilder`:
+
+```csharp
+public static MauiApp CreateMauiApp()
+{
+    var builder = MauiApp.CreateBuilder();
+    builder
+        .UseMauiApp<App>()
+
+        // Add this section anywhere on the builder:
+        .UseSentry(options =>
+        {
+            // The DSN is the only required setting.
+            options.Dsn = "___PUBLIC_DSN___";
+
+            // Use debug mode if you want to see what the SDK is doing.
+            // Debug messages are written to stdout with Console.Writeline,
+            // and are viewable in your IDE's debug console or with 'adb logcat', etc.
+            // This option is not recommended when deploying your application.
+            options.Debug = true;
+
```

Comment:
Even though we currently don't send any transactions auto of the box, if users already add this, once we add auto transactions, it'll already work automatically. Then we can also avoid adding it below in the perf step:

```suggestion
            // Set TracesSampleRate to 1.0 to capture 100% of transactions for performance monitoring.
            // We recommend adjusting this value in production.
            options.TracesSampleRate = 1.0;
```